mkdocs/themes/readthedocs/breadcrumbs.html

```diff
@@ -13,11 +13,13 @@
     {% if page %}<li>{{ page.title }}</li>{% endif %}
     <li class="wy-breadcrumbs-aside">
       {%- block repo %}
-      {% if repo_url %}
-        {% if repo_name == 'GitHub' %}
-          <a href="{{ repo_url }}" class="icon icon-github"> Edit on GitHub</a>
+      {% if page and page.edit_url %}
+        {% if repo_name == GitHub %}
+          <a href="{{ page.edit_url }}" class="icon icon-github"> Edit on {{ repo_name }}</a>
         {% elif repo_name == 'Bitbucket' %}
-          <a href="{{ repo_url }}" class="icon icon-bitbucket"> Edit on BitBucket</a>
+          <a href="{{ page.edit_url }}" class="icon icon-bitbucket"> Edit on {{ repo_name }}</a>
+        {% else %}
+          <a href="{{ page.edit_url }}"> Edit on {{ repo_name }}</a>
```

Comment:
> keeping the template changes to a minimum

I figured as much. Not a big deal, but with the current changes, if makes sense to clean it up now.